web-ui/src/main/resources/catalog/components/edit/crsselector/CRSSelector.js

```diff
@@ -31,6 +31,12 @@
              scope.snippetRef = gnEditor.
              buildXMLFieldName(scope.elementRef, scope.elementName);
 
+             // Replace the name attribute with id since this textarea is used only to store the template, we don't wanna submit it
```

Comment:
Line is probably too long, you could use ```mvn clean install -Pjslint``` to fix formatting (see https://github.com/geonetwork/core-geonetwork/tree/develop/web-ui#check--format-file)